cupy/indexing/generate.py

```diff
@@ -1,13 +1,167 @@
 # flake8: NOQA
 # "flake8: NOQA" to suppress warning "H104  File contains nothing but comments"
 
-
-# class c_(object):
-# class r_(object):
 # class s_(object):
 
 import numpy
+
 import cupy
+
+import six
+
+
+class AxisConcatenator(object):
```

Comment:
CClass specifies all arguments and RClass uses the default values. This implementation looks strange.